### ISSUE TITLE
Using with FlowRouter

### ISSUE THREAD

**Issue by nicooprat** [Akryum/meteor-vue-component#58]
I'm having a hard time to use this package with FlowRouter. I'd like to keep it instead of using `vue-router` to keep maximum integrity with the Meteor ecosystem (such as `useraccounts:flow-routing`).

What would be the right way to update components on route change?

I saw this [kickstart-meteor-vue-huge-app repo](https://github.com/Akryum/kickstart-meteor-vue-huge-app/tree/66ad3332c9f62e3fbc6b521b521739ada578a988/app/modules/todos/client) but I couldn't replicate your workflow and it seems a bit overkill for such a simple thing...

Thanks for your help.

**Comment by Akryum** [comment]
Well, you can still take a look at this project and more precisely at these files:
- [module.js](https://github.com/Akryum/kickstart-meteor-vue-huge-app/blob/66ad3332c9f62e3fbc6b521b521739ada578a988/app/main/client/imports/module.js) where the dynamic component swapping is implemented,
- [module/Module.vue](https://github.com/Akryum/kickstart-meteor-vue-huge-app/blob/66ad3332c9f62e3fbc6b521b521739ada578a988/app/modules/todos/client/components/Module.vue) is the Vue component with the `<component :is="view"></component>` special directive,
- [module/module.js](https://github.com/Akryum/kickstart-meteor-vue-huge-app/blob/66ad3332c9f62e3fbc6b521b521739ada578a988/app/modules/admin/client/module.js) is a subclass to [module.js](https://github.com/Akryum/kickstart-meteor-vue-huge-app/blob/66ad3332c9f62e3fbc6b521b521739ada578a988/app/main/client/imports/module.js) where you tell which component to use as the layout and which one to embed depending on the route,
- [module/routes.js](https://github.com/Akryum/kickstart-meteor-vue-huge-app/blob/66ad3332c9f62e3fbc6b521b521739ada578a988/app/modules/todos/client/routes.js) where you define flow-router routes.

To sum up, you need to implement the dynamic component switching yourself using `<component :is="compName"></component>`.

**Comment by nicooprat** [comment]
Ok, well, I may try it some day! Thanks for your answer.
